sale_financial_risk/models/sale.py

```diff
@@ -0,0 +1,137 @@
+# Copyright 2016-2020 Tecnativa - Carlos Dauden
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import _, api, fields, models
+from odoo.tools import float_round
+
+
+class SaleOrder(models.Model):
+    _inherit = "sale.order"
+
+    # Index this field is affected by the related field risk_partner_id. Mainly when
+    # commercial fields in the partner are written and thus recomputation of that
+    # relation is triggered.
+    partner_invoice_id = fields.Many2one(index=True)
+
+    def evaluate_risk_message(self, partner):
+        self.ensure_one()
+        risk_amount = self.currency_id._convert(
+            self.amount_total,
+            partner.risk_currency_id,
+            self.company_id,
+            self.date_order
+            and self.date_order.date()
+            or fields.Date.context_today(self),
+            round=False,
+        )
+        exception_msg = ""
+        if partner.risk_exception:
+            exception_msg = _("Financial risk exceeded.\n")
```

Comment:
```suggestion
            exception_msg = self.env._("Financial risk exceeded.\n")
```

Can you do this change in all occurrences ?

Reply:
Done